docs/eks-installation.md

```diff
@@ -71,7 +74,8 @@ This also deploys `cert-manager v1.8.2` and `Antrea v1.8`.
 ### Deploy Nephe Controller
 
 To deploy the latest version of nephe (built from the main branch), use the
-checked-in [deployment yaml](../config/nephe.yml):
+checked-in [deployment yaml](../config/nephe.yml) after `cert-manager` Pods are
```

Comment:
As you are making the change there, can you update the doc to show release version too.. Check this https://github.com/antrea-io/antrea/blob/main/docs/eks-installation.md.

Reply:
Sure we can do that too with other minor fixes in review. can you do that today?

Reply:
this is done as a part of #21